client/cmdhf14a.h

```diff
@@ -12,7 +12,19 @@
 #ifndef CMDHF14A_H__
 #define CMDHF14A_H__
 
+#include <stdio.h>
 #include <stdint.h>
+#include <stdlib.h>
+#include <inttypes.h>
+#include <string.h>
+#include <unistd.h>
+#include <stdbool.h>
+
```

Comment:
why are we moving includes to the .h from the .c ?  

just curious.  (i know code style can be argued but...)